lms/static/scripts/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2018,
```

Comment:
I set the version to 2018 because the code makes use of [Object Rest/Spread](https://nitayneeman.com/posts/object-rest-and-spread-properties-in-ecmascript-2018/) in a few places.